packages/apps/api/src/handlers/auth.ts

```diff
@@ -11,6 +11,11 @@ export async function validateCredentials(auth0Client: Auth0Client, request: Req
     response.status(400).json(toMessage("All fields must be provided and non-blank"));
   }
 
+  const userFetchResult = await auth0Client.getUserByEmail(email);
+  if (userFetchResult.status != ResponseStatus.Success) {
+    return errorResponse(userFetchResult.message, userFetchResult.status);
```

Comment:
I think this is a copy-paste error? `return errorResponse` returns an error object. We want to do something like `response.status(404).json(userFetchResult.message)` here I think?